.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[TITLE]"
+labels: 'bug-report'
```

Comment:
Why a label and a name that repeats the same information?

Reply:
AFAICT, when an issue is created, it will asked for this info inside the `---` delimeters, and then auto-populate the fields after them.